puppetboard/templates/index.html

```diff
@@ -7,7 +7,6 @@
 {% if config.DAILY_REPORTS_CHART_ENABLED %}
 {% block script %}
 <script src="{{url_for('static', filename='js/d3.min.js')}}"></script>
-<script src="{{url_for('static', filename='js/c3.min.js')}}"></script>
```

Comment:
I removed this because on line 4-6 we have:

```
{% if config.DAILY_REPORTS_CHART_ENABLED %}
<link href="{{ url_for('static', filename='css/c3.min.css') }}" rel="stylesheet">
{% endif %}
```

Should `d3.js` be in the block as well, or is it used elsewhere?

Reply:
Still waking up I shouldn't have removed this was mixing js and css up.